### PR TITLE
Baklava in Swift

### PR DESCRIPTION
**Pull request by mdmagahis**
- [ ] I would **NOT** like credit for my contribution in the article (credit will be included otherwise)
- [ ] I plan to write the article for this code snippet
- [x] I named the pull request using `<Sample Program> in <Language>` format
- [ ] I created/updated the language README
  - [x] I added the sample program name to the README
  - [ ] I added fun facts (i.e. debut, developer, typing, etc.)
  - [ ] I added reference link(s) to the README
  - [ ] I added solution citations when necessary (see [plagiarism][2])
- [ ] I fixed #your-issue-number-here

## Notes

First Pull Request here. Please let me know what I need to work on.

[1]: https://therenegadecoder.com/code/sample-programs-in-every-language/
[2]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/CONTRIBUTING.md#plagiarism

### CODE REVIEW

**Comment by jrg94** [comment]
If available, I'm going to have @martyav review this because they're our resident expert in Swift. Otherwise, someone in @TheRenegadeCoder/core will take a look.

File: archive/s/swift/README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Welcome to Sample Programs in Swift!
 - [Hello World in Swift](https://therenegadecoder.com/code/hello-world-in-swift/)
 - [Reverse String in Swift](https://therenegadecoder.com/code/reverse-a-string-in-swift/)
 - [Fizzbuzz in Swift](https://github.com/TheRenegadeCoder/sample-programs/issues/482)
+- Baklava in Swift
```
Comment (jrg94) [inline archive/s/swift/README.md]: Can you create an article issue like #32 and link it here?

**Comment by jrg94** [comment]
Hey @mdmagahis! Just checking in.

**Comment by mdmagahis** [comment]
@jrg94 does #620 look right?

**Comment by jrg94** [comment]
Looks great! You just need to link it in the README now.

**Pull request merged**